packages/perseus/README.md

```diff
@@ -0,0 +1,3 @@
+# @khanacademy/perseus
+
+This is the main package! Import this to render perseus items. Currently, UI that's outside of a perseus item (check answer button, anything at the higher-than-assessment-item level) is left as an exercise for the client.
```

Comment:
This is a decent start.  I think one thing to keep in mind is that this readme also becomes the text used at https://www.npmjs.com/package/@khanacademy/perseus 

With that in mind, I think it'd be useful to introduce Perseus itself in the first line. Maybe something like: 

"Perseus is the set of components that we use at Khan Academy to power our Exercises and Articles."